DataSource/DataSource/Sources/Interface/NearbyNetworkInterface.swift

```diff
@@ -53,8 +50,43 @@ public protocol NearbyNetworkConnectionDelegate: AnyObject {
     ///   - connections: 검색된 기기들
     func nearbyNetwork(_ sender: NearbyNetworkInterface, didFind connections: [NetworkConnection])
 
+    /// 검색된 기기가 사라졌을 때 실행됩니다.
+    /// - Parameters:
+    ///   - connection: 사라진 기기
+    func nearbyNetwork(_ sender: NearbyNetworkInterface, didLost connection: NetworkConnection)
+
     /// 주변 기기와의 연결에 실패했을 때 실행됩니다.
     func nearbyNetworkCannotConnect(_ sender: NearbyNetworkInterface)
+    
+    /// 주변 기기와 연결에 성공하였을 때 실행됩니다.
+    /// - Parameters:
+    ///   - connection: 연결된 기기
+    ///   - info: 기존 Session 정보
+    func nearbyNetwork(
+        _ sender: NearbyNetworkInterface,
+        didConnect connection: NetworkConnection,
+        with info: [String: String])
+
+    /// 주변 기기와 연결에 성공했을 때 실행됩니다. 
+    /// - Parameters:
+    ///   - context: 참여자가 보낸 정보
+    ///   - isHost: 호스트 여부
+    func nearbyNetwork(
+        _ sender: NearbyNetworkInterface,
+        didConnect connection: NetworkConnection,
+        with context: Data?,
+        isHost: Bool
+    )
```

Comment:
단순 스타일입니다만, 아래가 컨벤션에 맞습니다~!
```suggestion
        isHost: Bool)
```

Reply:
앗 프로토콜에서도 선언인데, 중괄호 없으니 소괄호가 올라오나요??
선언이라 생각해서 아래로 내려가야한다고 생각하고 있었는데..

Reply:
헉 저희 저번에 일단 올리는 것으로 통일하기로 했던걸로 기억합니다!! 
컨벤션 깃 링크에 해당 내용은 확인하지 못해서, 중괄호 없으면 올리고 있으면 내리는걸로 현상유지하기로 했습니다!

Reply:
일단 올려서 푸시했습니다!